Task2/js/app.js

```diff
@@ -1,13 +1,19 @@
 document.addEventListener('DOMContentLoaded', () => {
+    ZOMBIE_TYPE.DEAD = 0;
+
     const zombieList = document.querySelector('.zombies');
 
-    if (Array.isArray(zombies)){
-        for (let i = 0; i < zombies.length; i++) {
-            renderZombie(zombies[i], i);
+    renderZombieList(zombies);
+
+    function renderZombieList(units) {
+        zombieList.innerHTML = '';

```

Comment:
well ... it's not a good idea to remove the all zombies on every zombie's dead

Reply:
done

Reply:
done